talks/broad_12_mar_2018.md

```diff
@@ -0,0 +1,558 @@
+
+3. All nodes and edges are ancestral to at least one sample.
+
+4. No adjacent redundant edges 
+    (e.g., $(\ell, x, p, c) + (x, r, p, c) \rightarrow (\ell, r, p, c)$).
+
+-----------
+
+*Answer:* to simplify a tree sequence
+to the history of the *samples*:
+
+1. Paint each *sampled* chromosome a distinct color.
+
+2. Moving back up the tree sequence,
+    copy colors of each chromosome to the parental chromosomes
+    they inherited from.
+
+3. If two colors go in the same spot (*coalescence*),
+    replace with a new color (unique to that ancestor).
+    Output a node for the ancestor and an edge for the coalescence.)
+
+4. Once all colors have coalesced in a given segment,
+    stop propagating it.
+
+## An example: simplify these to J and K
+
+![Simplify example](simplify_walkthrough.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.0.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.1.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.2.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.3.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.4.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.5.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.6.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.7.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.8.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.9.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.10.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.11.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.12.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.13.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.14.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.15.png){ width="100%" }
+
+
+## Wright-Fisher, N=10: before simplification
+
+![Wright-Fisher tree sequence](sim_wf.anim.gif)
+
+## Wright-Fisher, N=10: before simplification
+
+![Wright-Fisher tree sequence](sim_wf_unlabeled.anim.gif)
+
+## ... and after simplification
+
+![Simplified Wright-Fisher tree sequence](sim_wf_simplified.anim.gif)
+
```

Comment:
for talking about these, maybe it would be nice to see them side-by-side. not sure if it'd render well...

Reply:
too much, i suspect